napari_allencell_annotator/widgets/file_item.py

```diff
@@ -67,8 +67,11 @@ def get_file_path_str(self) -> str:
         return str(self._file_path)
 
     def get_name(self) -> str:
-        """Return basename"""
-        return self._file_path.stem
+        """Return parent file name for zarr. Otherwise, return file name."""
+        if self._file_path.suffix == ".zarr":
+            return self._file_path.parent.stem
+        else:
+            return self._file_path.stem
```

Comment:
lets also refactor this and move into fileutils

Reply:
finished in https://github.com/aics-int/napari-allencell-annotator/pull/79/commits/20ede01654b3f694ef0409a53d13d5780e3872e3